<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite ignored="3" hostname="DESKTOP-QM6IKUM" failures="0" tests="3" name="testcases.TC001_LoginAndLogout" time="0.001" errors="1" timestamp="12 Sep 2021 12:44:02 GMT" skipped="2">
  <testcase classname="testcases.TC001_LoginAndLogout" name="loginAndLogout" time="0.000">
    <skipped/>
  </testcase> <!-- loginAndLogout -->
  <testcase classname="testcases.TC001_LoginAndLogout" name="loginAndLogout" time="0.001">
    <skipped/>
  </testcase> <!-- loginAndLogout -->
  <testcase classname="testcases.TC001_LoginAndLogout" name="beforeMethod" time="0.000">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at wdMethods.SeMethods.takeSnap(SeMethods.java:367)
	at utils.Reporter.reportStep(Reporter.java:43)
	at utils.Reporter.reportStep(Reporter.java:64)
	at wdMethods.SeMethods.startApp(SeMethods.java:86)
	at wdMethods.ProjectMethods.beforeMethod(ProjectMethods.java:36)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:590)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </error>
  </testcase> <!-- beforeMethod -->
  <testcase classname="wdMethods.ProjectMethods" name="afterTest">
    <skipped/>
  </testcase> <!-- afterTest -->
  <testcase classname="wdMethods.ProjectMethods" name="beforeSuite">
    <skipped/>
  </testcase> <!-- beforeSuite -->
  <testcase classname="wdMethods.ProjectMethods" name="afterSuite">
    <skipped/>
  </testcase> <!-- afterSuite -->
</testsuite> <!-- testcases.TC001_LoginAndLogout -->
